core/nodepool/config/config.go

```diff
@@ -1,6 +1,6 @@
 package config
 
-//go:generate go run ../../codegen/templates_gen.go DefaultClusterConfig=cluster.yaml StackTemplateTemplate=stack-template.json
+//go:generate go run ../../../codegen/templates_gen.go DefaultClusterConfig=cluster.yaml StackTemplateTemplate=stack-template.json
 //go:generate gofmt -w templates.go
```

Comment:
Now these files have moved, we should update [.gitignore](https://github.com/coreos/kube-aws/blob/master/.gitignore#L5) to match. I suggest `**/config/templates.go`.